docs/section23/Rule23-11.md

```diff
@@ -0,0 +1,53 @@
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.17 System 11 Supply Air Temperature and Fan Control  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-11.  
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. baseline_system_type_compare()
+
+**Applicability Checks:**  
+- create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_11]`
+- Get B-RMR system types: `baseline_system_types_dict = get_baseline_system_types(B-RMR)`
+
+- loop through the applicable system types: `for target_sys_type in APPLICABLE_SYS_TYPES:`
+    - and loop through the baseline_system_types_dict to check the system types of the baseline systems: `for system_type in baseline_system_types_dict:`
+        - do baseline_system_type_compare to determine whether the baseline_system_type is the applicable_system_type: `if((baseline_system_type_compare(system_type, target_sys_type, false)):`
+            - the systems in the list: baseline_system_types_dict[system_type] are sys-11 - loop through the list of systems: `for hvac_system_id in baseline_system_types_dict[system_type]:`
+            - `CONTINUE TO RULE LOGIC`
+            - otherwise, rule not applicable: `else: RULE_NOT_APPLICABLE`
+            
+  **Rule Logic:**
+  - create boolean for whether rule passes: `rule_passes = TRUE`
+  - get the hvac system: `hvac_system = get_object_by_id(hvac_system_id)`
+  - get the fan_system: `fan_system = hvac_system.fan_system`
+  - check whether the fan_system.temperature_control is LOAD_RESET_TO_SPACE_TEMPERATURE, other reset options do not result in correct control: `if fan_system.temperature_control == LOAD_RESET_TO_SPACE_TEMPERATURE:`
+      - get the fan_system reset_differential_temperature - this temperature is added to the terminal.supply_design_cooling_setpoint temperature to give the supply air temperature at minimum cooling load: `reset_diff_temp = fan_system.reset_differential_temperature`
+      - we now need to check each zone and terminal for compliance - look through each zone: `for zone in B-RMR...zones:`
+          - get the zone design cooling setpoint: `zone_cooling_setpoint = zone.design_thermostat_cooling_setpoint`
+          - look at each terminal and check compliance if the terminal is connected to system 11: `for terminal in zone.terminals:`
+              - check to see if the terminal is connected to the hvac_system: `if terminal.served_by_heating_ventilating_air_conditioning_system == hvac_system_id:`
+                  - check to see if the reset_diff_temp added to the terminal.supply_design_cooling_setpoint_temperature DOES NOT equal the zone_cooling_setpoint, set rule_passes to FALSE: `if((reset_diff_temp + terminal.supply_design_cooling_setpoint_temperature) !=  zone_cooling_setpoint): rule_passes = FALSE`
```

Comment:
discussed on 12/7/2023 call - @claperle noticed that this rule (which references a section of G3.1.3.17), conflicts with G3.1.3.12.  Upon examining the two, it seems likely that the inclusion of System 11 in G3.1.3.12 was an error.  We are keeping this rule as non-mandatory because of this conflict, and because of my concerns listed in my [comment above](https://github.com/pnnl/ruleset-checking-tool/pull/1173#issuecomment-1832597599)